### PR TITLE
fix: update transfer-05-file-transfer-cloud sample

### PR DESCRIPTION
**Pull request by bpetrovbg**
## What this PR changes/adds

updates the provider/consumer configurations, builds and README for transfer-05-file-transfer-cloud

## Why it does that

Sample is out of date.

## Linked Issue(s)

Closes [Issue fetching Retrieve provider Contract Offers](https://github.com/eclipse-edc/Samples/issues/236)

### CODE REVIEW

File: transfer/transfer-05-file-transfer-cloud/cloud-transfer-consumer/build.gradle.kts [hunk 1]
```diff
@@ -36,6 +36,20 @@ dependencies {
     implementation(libs.edc.dsp)
     
     implementation(libs.edc.data.plane.selector.core)
+
+    //configuration from earlier examples
```
Comment (ndr-brt) [inline transfer/transfer-05-file-transfer-cloud/cloud-transfer-consumer/build.gradle.kts:40]: this is not "configuration" but these are dependencies, please update the comment accordingly (or removing it altogether, as it is not really necessary). same for the other build file

**Review by ndr-brt — COMMENTED** [review]
hi, thanks for contribution, please sign the ECA

File: transfer/transfer-05-file-transfer-cloud/cloud-transfer-consumer/build.gradle.kts (hunk 1 above)
Comment (bpetrovbg) [reply transfer/transfer-05-file-transfer-cloud/cloud-transfer-consumer/build.gradle.kts]: you are right, I just did a minor cleanup 😉

**Comment by ndr-brt** [comment]
@bpetrovbg please fix the PR name according to the convention https://eclipse-edc.github.io/docs/#/documentation/CONTRIBUTING
when the PR is ready please re-request my review so I can eventually approve and merge

**Comment by ndr-brt** [comment]
@bpetrovbg the PR name is still not valid: https://github.com/eclipse-edc/Samples/actions/runs/8875401453/job/24366181117?pr=247

**Review by ndr-brt — CHANGES_REQUESTED** [review]
@bpetrovbg the PR name is still not valid: https://github.com/eclipse-edc/Samples/actions/runs/8875401453/job/24366181117?pr=247

**Review by ndr-brt — APPROVED** [review]

**Pull request merged**